reana/reana_dev/git.py

```diff
@@ -569,12 +638,17 @@ def git_checkout(branch, fetch):  # noqa: D301
 @click.option(
     "--branch", "-b", nargs=2, multiple=True, help="Which PR? [component PR#]"
 )
+@click.option(
```

Comment:
What do you think about centralising this option [like `add_workflow_option` in REANA-Client](https://github.com/reanahub/reana-client/blob/577e5dd52f14a75bed32863ccbd419e6c2ebac0d/reana_client/cli/utils.py#L71)?

Do you think it would make sense to add the default branch value to config? I don't see why we would change it ever.. but at least it would be centralised.

Reply:
All updated. I have also improved reporting in case base branch does not exist in origin/upstream. Example: 

```console
$ reana-dev git-status -c r-a-vomsproxy -s --base maint-0.7
ERROR: Branch upstream/maint-0.7 does not exist.
reana-auth-vomsproxy @ master @ a315022 release: v1.0.0
```